components/AnimatedImage/library/src/desktopMain/kotlin/org/jetbrains/compose/animatedimage/DesktopAnimatedImage.kt

```diff
@@ -26,42 +26,50 @@ actual suspend fun loadResourceAnimatedImage(path: String): AnimatedImage {
 
 @Composable
 actual fun AnimatedImage.animate(): ImageBitmap {
-    val transition = rememberInfiniteTransition()
-    val frameIndex by transition.animateValue(
-        initialValue = 0,
-        targetValue = codec.frameCount - 1,
-        Int.VectorConverter,
-        animationSpec = infiniteRepeatable(
-            animation = keyframes {
-                durationMillis = 0
-                for ((index, frame) in codec.framesInfo.withIndex()) {
-                    index at durationMillis
-                    val frameDuration = calcFrameDuration(frame)
-
-                    durationMillis += frameDuration
-                }
+    when (codec.frameCount) {
+        0 -> return ImageBitmap(0, 0) // No frames at all
```

Comment:
Compose have a bug (or a strange feature), when we use bitmaps with zero size:
```
Exception in thread "main" java.lang.IllegalStateException: Size is unspecified
	at androidx.compose.ui.geometry.Size.getWidth-impl(Size.kt:48)
	at androidx.compose.ui.draw.PainterModifier.draw(PainterModifier.kt:267)
	at androidx.compose.ui.node.DrawEntity.draw(DrawEntity.kt:98)
	at androidx.compose.ui.node.LayoutNodeWrapper.drawContainedDrawModifiers(Layout
```

Use `ImageBitmap.Blank` here, which has size = 1, until it will be fixed